ebu_tt_live/node/consumer.py

```diff
@@ -76,7 +76,7 @@ class ReSequencer(AbstractProducerNode, SimpleConsumer):
     _expects = EBUTT3Document
     _provides = EBUTT3Document
 
-    def __init__(self, node_id, reference_clock, segment_length, discard, sequence_identifier,
+    def __init__(self, node_id, reference_clock, segment_length, init_document, discard, sequence_identifier,
```

Comment:
Is `init_document` required? If not, we should set a default `=None` for it. It is not enough to rely on the default from the config read by `ebu-run`, because this class can be instantiated by a script separately from that config mechanism.

Reply:
You're right; I therefore added a default value in 9fbe83f37ac74ca4a888f7143a3fe3fabd681c4d.